Bottom: 4e71985c9b1dd1ad587c58a6d3fef640f62461ab
Top:    fc95c4f361459da6e82be1ac9ce3f2e14944b5c9
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:14:33 -0700

fixup-i-dont-know-where


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 01bd59a0512..23cd37cb503 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -548,7 +548,6 @@ class FileHandleTestCase(lldbtest.TestBase):
 
     @add_test_categories(['pyapi'])
     @skipIf(py_version=['<', (3,)])
-    @skipIf(True) # fixme multiple problems with this
     def test_string_out(self):
         f = io.StringIO()
         status = self.debugger.SetOutputFile(f)
